libs/libsemanage/Makefile

```diff
@@ -0,0 +1,99 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libsemanage
+PKG_VERSION:=3.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/selinux/releases/download/20200710
+PKG_HASH:=22d6c75526e40d1781c30bcf29abf97171bdfe6780923f11c8e1c76a75a21ff8
+PKG_BUILD_DEPENDS:=PACKAGE_python-libsemanage:python3 PACKAGE_python-libsemanage:swig/host
+HOST_BUILD_DEPENDS:=audit/host libselinux/host bzip2/host
+
+PKG_MAINTAINER:=Thomas Petazzoni <thomas.petazzoni@bootlin.com>
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/libsemanage
+  SECTION:=libs
+  DEPENDS:=+libaudit +libselinux +libbz2
+  CATEGORY:=Libraries
+  TITLE:=SELinux policy management library
+  URL:=http://selinuxproject.org/page/Main_Page
+endef
+
+define Package/python-libsemanage
+  TITLE:=Python bindings for the SELinux policy management library
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  DEPENDS:=+python3 +libsemanage
+endef
+
+define Package/libsemanage/description
+	libsemanage is the policy management library. It uses
+	libsepol for binary policy manipulation and libselinux for
+	interacting with the SELinux system. It also exec's helper
+	programs for loading policy and for checking whether the
+	file_contexts configuration is valid (load_policy and
+	setfiles from policycoreutils) presently, although this may
+	change at least for the bootstrapping case (for rpm).
+endef #'
+
+include $(INCLUDE_DIR)/host-build.mk
+
+HOST_MAKE_FLAGS += \
+	PREFIX=$(STAGING_DIR_HOSTPKG)
```

Comment:
Somehow the host-build still tries to create `/etc/selinux/semanage.conf` on the buildhost. (usual `/etc/` isn't inside `PREFIX`, so I guess setting PREFIX alone doesn't help)

Reply:
The latest push makes use of DESTDIR rather than PREFIX.